submissions/antonkottans/js-dom/style.css

```diff
@@ -0,0 +1,78 @@
+.body {
+	margin: 0px;
+	display: grid;
+	grid-template-areas: "nav main" "nav main";
+	grid-template-columns: 25% 75%;
+    grid-template-rows: 50% 1fr;
```

Comment:
Strange indentations.

Reply:
there were spaces and tabs in different lines, don't know why, in vsc all were good, fixed